test.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+	"extends": "eslint-config-leankit/base",
+	"env": {
+		"browser": true,
+		"node": true,
+		"mocha": true
+	},
+	"globals": {
+		"before": true,
+		"beforeEach": true,
+		"describe": true,
+		"global": true,
+		"it": true,
+		"sinon": true,
+		"should": true
+	},
+	"rules": {
+		"max-nested-callbacks": [ 2, 15 ],
```

Comment:
Here is an example where we bumped up the callback level for tests since mocha test groups can get pretty deep.